source/javascripts/embertimes-test-ids.js

```diff
@@ -0,0 +1,27 @@
+$(function(){
+  if ($('[data-embertimes-page]').length > 0) {
+    applyEmberTimesTestIds();
+  }
+
+
+   function applyEmberTimesTestIds() {
+     let sectionTitles = $('[data-embertimes-post-body] h2');
+     sectionTitles.attr('data-embertimes-section-title', true);
+
+     sectionTitles.each((i, title) => {
+       $(title).attr('data-embertimes-section-title', i);
+       let sectionFragments = $(title).nextUntil('h2').not('hr');
+       sectionFragments.each((index, element) => {
+         $(element).attr('data-embertimes-section-fragment', i);
```

Comment:
Should this be:
```suggestion
         $(element).attr('data-embertimes-section-fragment', index);
```

Reply:
I think for the collection of the sections with the script it's mostly important that all the parts that a section is composed of can be identified as belonging together to that one section. E.g. each section may have several `p` or `ul` elements that we want to group together and copy-paste into one single textfield in the goodbits template interface in the end

I was thinking to just use the id that identifies a section to make sure we can easily [fetch all the sub parts of the section in one go in the automated template generation script](https://github.com/ember-learn/ember-times-tools/pull/28/files#r293800115) and I thought being able to distinguish the separate sub paragraphs wouldn't be relevant for the current template creation workflow for now

Does this make sense? Feel free to let me know if I missed something here

Reply:
Yep, I see what you wanted to do now. Thanks!